run.ts

```diff
@@ -104,20 +103,20 @@ export async function runWithDeno({ reloadUrl, results, signal }: RunWithDenoOpt
   })
 }
 
-async function executionWrapper(fileName: string, run: (outputBuffer: Buffer) => Promise<number>) {
-  console.log(`running ${fileName}`)
+async function executionWrapper(filePath: string, run: (outputBuffer: Buffer) => Promise<number>) {
```

Comment:
Would be cleaner to...

A) add an explicit return type of `[string, string]`, the `as const`
B) below, remove the `as const` from the returned tuple

Reply:
sure